pkgs/development/libraries/gsettings-desktop-schemas/default.nix

```diff
@@ -29,9 +29,14 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     glib
+  ] ++ lib.optionals (stdenv.buildPlatform == stdenv.hostPlatform) [
     gobject-introspection
```

Comment:
This should likely go to native build inputs

Reply:
Yes, this wasn't fully correct. Should be better now.